digiwf-apps/packages/apps/digiwf-webcomponent/src/components/ServiceInstanceListItem.vue

```diff
@@ -14,13 +14,19 @@
     >
       {{ serviceInstance.definitionName }}
     </dynamic-heading>
-    <p class="mb-1">Erstellt am {{ createdDate }}</p>
     <p
       v-if="serviceInstance.endTime"
       class="mb-1"
     >
-      Abgeschlossen am {{ endedDate }}
+      {{ DefaultStatus.FINISHED }} am {{ endedDate }}
     </p>
+    <p
+      v-else
+      class="mb-1"
+    >
+      {{ DefaultStatus.STARTED }} am {{ createdDate }}
+    </p>
+    <p>Status: {{ statusText }}</p>
```

Comment:
Ich überlege gerade, ob das hier nicht angezeigt werden sollte, wenn der Prozess abgeschlossen ist. In deinem Bild zeigt es zwei Mal "Abgeschlossen". Sieht irgendwie nicht so schön aus.

Reply:
Hatte jetzt nur noch einen Mobiles Arbeiten Antrag, aber da steht z.B. `Antrag geprueft` und glaub bei den meisten realen Anträgen wird da ein aussagekräftigerer Status gesetzt. Kommt wahrscheinlich auf den Prozess an ...

Reply:
Ja genau, so bin ich auf drauf gekommen, dass der Status keinen Sinn macht. Wenn der Prozess keinen expliziten End-Status hat, der am Ende gesetzt wird, dann gilt der letzte Status und der kann, wie hier bei MobiA, nicht passen.

Reply:
Idee war, dass sich die Höhe des Elements dann nicht verändert, da weiterhin die Zeile mit dem (Abgeschlossen am xx.xx.xxxx)l + der Status angezeigt wird. Also immer zwei Zeilen. 
Kann es nicht mehrere Stati geben, die zu einer Terminierung eines Prozesses führen? Dann wäre es ja schon noch gut beides anzuzeigen, oder nicht?